### ISSUE TITLE
[question]insert anchor and link

### ISSUE THREAD

**Issue by soso4906** [zim-desktop-wiki/zim-desktop-wiki#2651]
Could someone please explain in detail how to insert an anchor within the same page and a hyperlink that links to the anchor?
I want to use an in-page link, so I updated to zim 0.75.2.
I've been struggling with it for two days, but I still don't know how to operate it.
For example, when I click on the string "see here" at the top of the page, I want it to jump to the heading "bbbb" at the bottom of the page.
I'm not good at English, so I'd appreciate an explanation with pictures.

**Comment by jaap-karssenberg** [comment]
The anchor can be typed as "##see-here", 2x # without white spaces, if
succesfull it is replaced by an small icon (picture which can be clicked).
To link type "#see-here" (1x #) if succesfull it becomes a blue link.

Hope this helps

Jaap

Op vr 9 aug. 2024 07:21 schreef soso4906 ***@***.***>:

**Comment by soso4906** [comment]
Thanks for the advice.
But unfortunately it's still not solved.
I operated as follows:
I created a "New Sub page" under Main:Puppy named "test-link", then typed "#see_here" at the top of that new page, and "##see_here" at the bottom.
However, when I typed "##see_here" and pressed the Enter key, no icon was displayed and the text I typed disappeared.(A line break is inserted but the line becomes blank.)
When I place the cursor over "#see_here"(blue link) at the top of the page in this state, it displays "Go to "Main:Puppy:test-link#see_here"", but when I click it it returns "ld "see_here" not found on the current page".
What am I doing wrong?
If I look at the "Edit" tab under "Edit" > "Settings",  the "Automatically turn identifiers starting with "#" into links" is checked.

**Comment by soso4906** [comment]
I'll correct this.
My environment is a ubuntu-based Puppy Linux (fossapup64), which is a bit unusual.
When I first tried to install zim-0.75.2 with apt command, I got an error about "dependency problems" and was unable to install it.
So, based on information on the web, I first installed _0.72.0.1 with the Puppy package manager(PPM), then downloaded zim-0.75.2.tar.gz, unzipped it, and copied the zim folder inside it to /usr/lib/python3/dist-packages/.
When I started zim after doing this, the zim version was 0.75.2, so I continued using it as is.
Following your advice, I re-examined this procedure, uninstalled 0.72.0.1 from the package manager, restarted the O/S, and then started zim, and just as you advised, when I entered ##see_here at the bottom of the page, a small icon was displayed, and when I clicked #see_here at the top of the page, it jumped to that icon.
This is exactly what I expected.
Once again, thank you for your advice.
